src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -0,0 +1,137 @@
+- `ParticipantRef` : The ParticipantRef code must be provided,
+- `StopProviderCode` : The code used to identify the provider in the stop_id generation.
+
+## File headers
+
+Each Netex file starts with a header containing some information generated by the program: 
+- `/PublicationDelivery/@version`: **x.y:FR-NETEX_nnnn-a.b-c** with
+  + `x.y`:  the version of Netex XSD of Netex (actually 1.09)
+  + `nnnn`:identification of the profile ("FRANCE", "COMMUN", "ARRET",
+"LIGNE", "RESEAU", "HORAIRE", "CALENDRIER" or "TARIF") 
+  + `a.b` is the profile version (2.1)
+  + `c` is the number of the local implementation (containing only integers and `.`). See at the top of the document.
+- `/PublicationDelivery/PublicationTimestamp`: generation date of the Netex Feed using ISO8601,
+- `/PublicationDelivery/ParticipantRef`: The value of the corresponding parameter to the conversion action.
+
+## arrets.xml
+
+A `stop_area` is considered monomodal if all the trips having stop_times referencing any of its stop_points have a physical_mode of the same "mode" Netex.
+
+### Quay
+
+Each `Quay` node corresponds to a NTFS `stop_point`.
+
+Netex field | NTFS file | NTFS field | Note
+--- | --- | --- | --- | ---
+Quay/@id | stops.txt | stop_id | see (1) below.
+Quay/@version | | | fixed value `any`.
+Quay/PublicCode | stops.txt | stop_code | This node may not be prensent if the stop_point has no `stop_code`.
+Quay/Name | stops.txt | stop_name | 
+Quay/Centroid/Location | stops.txt | stop_lat and stop_lon | see (2) below
+SiteRef | stops.txt | parent_station | If the parent_station is multimodal, see (3) below.
+TransportMode | | see (3) below
+tariffZones | stops.txt | fare_zone_id | The fare zone is prefixed by the `ParticipantRef` prefix with a `:` separator
+AccessibilityAssessment | stops.txt | equipment_id | This node is present only if the `equipment_id` is specified. see [AccessibilityAssessment]() below.
+
+**(1) definition of id**
+The id is composed of several parts separated by `:`: 
+- a country code (ISO 3166-1). `FR` for France.
+- a city code (5 character INSEE code for France). Eventually, the city disctrict can be specified with one or two character separated by `-`. For exemple `75056-12`.
+- object type using an enumarate:
+  + `ZE` (ZONE D’EMBARQUEMENT), 
+  + `LMO` (LIEU D’ARRET MONOMODAL), 
+  + `PM` (POLE MONOMODAL), 
+  + `LMU` (LIEU D’ARRET MUTIMODAL), 
+  + `AC` (ACCES)
+- technical code of the stop
+- code of the provider of the technical stop or `LOC` if it's beeing attributed. 
+
+In this version of the connector:
+- the country code will be set to the fixed value `FR`,
+- the city code will be set to the fixed value `XXXXX`,
+- the technical code of the stop will contain the value of `stop_id`, with a replacement of `:` by `_` to avoid conflict between separators,
+- the code of the provider will be set to the value of `StopProviderCode` param (is specified) or  `LOC` otherwise.
+
+
+**(3) definition of SiteRef**
+TBD
```

Comment:
in the formatted view `TDB` does not appear in a new line.

Reply:
fixed